.gitmodules

```diff
@@ -22,3 +22,6 @@
 [submodule "src/monogame/external/vma"]
 	path = native/monogame/external/vma
 	url = https://github.com/LunarG/VulkanMemoryAllocator.git
+[submodule "ThirdParty/MonoGame.Templates"]
+	path = ThirdParty/MonoGame.Templates
```

Comment:
can we not put it in ThirdParty? Its not a third part its us. 
maybe `external` or something like that.

Reply:
ThirdParty is the same to me, if you have a better folder idea to put it in that would be good 👍